server/ctrlsubsonic/handlers_playlist.go

```diff
@@ -123,13 +133,22 @@ func (c *Controller) ServeUpdatePlaylist(r *http.Request) *spec.Response {
 		FirstOrCreate(&playlist)
 
 		// update meta info
-	playlist.UserID = user.ID
+	if playlist.UserID == 0 {
+		playlist.UserID = user.ID
+	} else {
+		if playlist.UserID != user.ID {
+			return spec.NewResponse()
+		}
+	}
```

Comment:
could we just do this?

```suggestion
	if playlist.UserID != 0 && playlist.UserID != user.ID {
		return spec.NewResponse()
	}
	playlist.UserID = user.ID
```